### DISCUSSION TITLE
Microsoft Snap assist support

### DISCUSSION THREAD

**Discussion by sangorrin** [rxhanson/Rectangle#1207] [Ideas]
Hi, it would be great if the tool supported the behavior of Microsoft Snap Assist (https://support.microsoft.com/en-us/windows/snap-your-windows-885a9b1e-a983-a3b1-16cd-c531795e6241#WindowsVersion=Windows_10).
- Let you choose another window after you snap the first
- Let you resize the layout using the dividing lines

<img width="705" alt="image" src="https://github.com/rxhanson/Rectangle/assets/1510584/ac58bbb6-30ad-4e40-a707-6b2489aa1a39">

**Comment by rxhanson** [comment]
Thanks for the detailed feedback. This is on my backlog for Rectangle Pro. I'm currently working on a library for a grid style display and have plans to use that to construct this feature when it is done.
